src/Symfony/Component/Validator/Constraints/Email.php

```diff
@@ -55,6 +55,7 @@ class Email extends Constraint
      */
     public $strict;
     public $mode;
+    public $normalizer;
```

Comment:
Don't we need to at least check that the `$normalizer` is indeed a callable as we are already validating the other options?

Reply:
Hi @fabpot. I've added the checks.